### ISSUE TITLE
The model results vary greatly

### ISSUE THREAD

**Issue by zronly** [sujoyp/wtalc-pytorch#10]
Hello, when i ran the pytorch source code(learning rate=0.0001, dataset=Thumos14reduced), I ran into a problem.
I ran ten times, but the results of several of them were quite different. When IoU=0.5, some can reach about 25%, but some can't reach it, only about 24% .what caused this?  Is it caused by randomness?

**Comment by sujoyp** [comment]
We did not notice variations of more than 0.3-0.5%. Anyway, this codebase should produce a bit better results (more than 24%) compared to what we reported in the paper (22.8%).
